Dockerfile

```diff
@@ -1,3 +1,4 @@
+# @TODO this file is to be removed: it only exists for reference, until we can get static compilation to work in Nix too
```

Comment:
To be fixed before merging, IMO, but my nix-fu is too weak today, and I need to figure out how to get `-musl` builds running :)

Reply:
Ref: https://old.reddit.com/r/NixOS/comments/f0yi3b/how_to_build_a_simple_static_rust_binary_using/